### PR TITLE
board/qemu-i386: some cleanup in file headers

### PR DESCRIPTION
**Pull request by haukepetersen**

### CODE REVIEW

File: boards/qemu-i386/include/board.h [hunk 1]
```diff
@@ -1,23 +1,15 @@
...
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public
- * License along with this library; if not, write to the Free Software
- * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
```
Comment (LudwigOrtmann) [inline boards/qemu-i386/include/board.h]: Why are you changing this? It's not problematic...

Comment (haukepetersen) [inline boards/qemu-i386/include/board.h]: For consistency.

Comment (LudwigOrtmann) [inline boards/qemu-i386/include/board.h]: Discuss in https://github.com/RIOT-OS/RIOT/pull/2445#discussion_r24579024

Comment (haukepetersen) [inline boards/qemu-i386/include/board.h]: so what about this license substituion, it's LGPL for LGPL, that should be fine, shouldn't it?

Comment (LudwigOrtmann) [inline boards/qemu-i386/include/board.h]: I just find it random to change 3 of the 66 occurrences of this license header.

Comment (haukepetersen) [inline boards/qemu-i386/include/board.h]: you have to start somewhere... And as I touched the headers anyway, why not clean up the licenses while I was at it?

Comment (LudwigOrtmann) [inline boards/qemu-i386/include/board.h]: Fine, I won't ask you to revert this ;)

Comment (haukepetersen) [inline boards/qemu-i386/include/board.h]: thx :-) so you would ack this PR then?

Comment (LudwigOrtmann) [inline boards/qemu-i386/include/board.h]: Only if you change the commit message (bonus points for split the commit), so it does not claim this fixes the header - it only serves unification ;)

Comment (haukepetersen) [inline boards/qemu-i386/include/board.h]: that I can do

**Comment by haukepetersen** [comment]
split commits and changed PR title

**Comment by LudwigOrtmann** [comment]
ACK

**Comment by haukepetersen** [comment]
will merge when Travis is happy.

**Comment by haukepetersen** [comment]
and go.

**Pull request merged**